docs/release/trg-0/trg-1-06.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 1.06 - Administrator's Guide (User Assistance)
```

Comment:
What does "User Assistance" mean? I think we can omit that

Reply:
TRG 1.06 - Administrator`s Guide